.travis.yml

```diff
@@ -1,15 +1,14 @@
 language: node_js
 sudo: false
 node_js:
-  - '8'
-  - '10'
+- '8'
+- '10'
 script:
-  - npm run test-ci
-  - node bin/build-locales
-  - npm run build
-  - npm run lint
-  # run integration tests using an addons-linter binary in a production-like environment
-  - npm run test-integration:production
+- npm run test-ci
+- node bin/build-locales
+- npm run build
+- npm run lint
+- npm run test-integration:production
```

Comment:
Why did you change the indentation and remove the comment? Or is that maybe what the `travis` cli did?

Reply:
Travis CLI does that.

Reply:
I'll fix it.